source/project-management.rst

```diff
@@ -74,15 +74,34 @@ Examples include:
 
 1. ``feature/235_preserve_prep`` for :ual-re:`LD-Cool-P#235 <LD-Cool-P/issues/235>`
 2. ``hotfix/229_400_error`` for :ual-re:`LD-Cool-P#229 <LD-Cool-P/issues/229>`
+3. ``chore/242_gitignore`` for :ual-re:`LD-Cool-P#242 <LD-Cool-P/issues/242>`
 
 Note: Our branching model initially followed a ``git-flow`` workflow with
 features, hotfixes, and releases; however, we later moved away from that
 model and now use a GitHub flow workflow where all changes are merged into
-the ``master``/``main`` branch after review and testing.
+the ``main``/``master`` branch after review and testing.
 
+Pushing to a remote branch
+------------------------
+After updating files, we can push the changes to remote branch. It is important to push to a branch (not ``main``) so that a team member can review the changes over a pull request.
+use the following ``git`` commands:
+
+::
+
+   $ git add .
+   $ git commit -m "<message>"
+   $ git branch                    # list all the branches and * is the current branch
+   $ git push origin <branchname>  # push to a remote branch
```

Comment:
Below L94 add:
In accordance with git best practices, the commit message should be short but descriptive. Avoid general messages like "updated file.txt" when possible.

Reply:
@yhan818 Is there a reason you decided not to add this? If you don't think it's necessary, we can leave it out but I thought it might be good to be explicit